src/Styling/Home.css

```diff
@@ -126,6 +127,52 @@
   width: 44%;
 }
 
+/* Footer Section */
+.Footer_section {
+  display:flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #08080A;
+}
+
+.Footer_container_1,
+ .Footer_container_2 {
+  width: 80%;
+  margin: 10px;
+  display:flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  text-align: center;
+}
+
+.Footer_container_2 {
+  margin-top: 20px;
+  display:flex;
+}
+
+.Footer_section button {
+  width: 195px;
+  height: 65px;
+  margin: 50px auto;
+  outline: none;
+  background-color: transparent;
+  border: 1px solid #fff;
+  cursor: pointer;
+  position: relative;
```

Comment:
- Why does this button need a position of relative?
- looking at the “.About_image_border button {...}” button in the styles above, this and that button are very different. Per the mockup, the two buttons only differ in a couple things like border & color. Why not create a style for the shared styles between the buttons and target specific buttons for specific differences(i.e. color)...This will be especially useful when creating the rest of the pages, because per the mockup, all buttons have mostly the same styles.

Reply:
-Removed z-index and replaced it with position relative. z-index is no longer used and bug issue was resolved.